node/resolvers/auth/index.ts

```diff
@@ -4,57 +4,64 @@ import { ResolverError, UserInputError } from '@vtex/api'
 import http, { Method } from 'axios'
 import { parse, serialize } from 'cookie'
 
-import { headers as authHeaders, withAuthToken } from '../headers'
+import { withAuthToken } from '../headers'
 import paths from '../paths'
+import { GetLoginSessionsResponse } from './types'
+import mapLoginSessionsFromAPI from './mapLoginSessionsFromAPI'
 
 const E_PASS = 'Password does not follow specified format'
 const E_TOKEN = 'VtexSessionToken cookie is null'
 
-export async function makeRequest(
-  ctx: any,
-  url: any,
-  method: Method = 'POST',
+interface ParamsMakeRequest {
+  ctx: any
+  url: any
+  method?: Method
+  body?: any
+  vtexIdVersion?: string
+  authCookieAdmin?: string | null
+  authCookieStore?: string | null
+}
+
+export async function makeRequest<T = any>({
+  ctx,
+  url,
+  method = 'GET',
+  body,
   vtexIdVersion = 'store-graphql',
-  cookie: string | null = null
-) {
+  authCookieAdmin = null,
+  authCookieStore = null,
+}: ParamsMakeRequest) {
```

Comment:
Why isn't that a client at `node/clients/`

Reply:
It should be, but this seems to be a piece of legacy code. A bigger step would be refactoring it to be at `node/clients`, but I only tried to improve it a little bit for now. That'll make the next step easier